dmapiclient/audit.py

```diff
@@ -69,6 +69,9 @@ class AuditTypes(Enum):
     # Mailing list actions
     mailing_list_subscription = "mailing_list_subscription"
 
+    # Buyer Email Domains
+    create_buyer_email_domain = "create_buyer_email_domain"
```

Comment:
Maybe this should sit under "Admin actions"?

Reply:
That is an excellent idea.